sh/tests/solutions/set-env-vars.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+export MY_MESSAGE="Hello World"
+export MY_NUM=100
+export MY_PI=3.142
+export MY_ARR=(one, two, three, four, five)
```

Comment:
you didn't ask the student to set this env var in the readme file!

Reply:
fixed